src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/dto/VilkårperiodeDto.kt

```diff
@@ -142,6 +143,11 @@ data class VilkårperioderDto(
     val aktiviteter: List<VilkårperiodeDto>,
 )
 
+data class VilkårperioderResponse(
+    val vilkårperioder: VilkårperioderDto,
+    val grunnlag: VilkårperioderGrunnlag,
```

Comment:
Burde vi ha en DTO for grunnlaget som sendes til frontend?

Reply:
Kanskje? Er jo bare json som vil være lik for begge, men kan være lurt hvis vi vil kunne gjøre endringer.

Reply:
Ja, kan være fint å separere modell som lagres i basen og det som sendes til frontend